web/ui/react-app/src/Navbar.tsx

```diff
@@ -25,6 +25,7 @@ const Navigation: FC<PathPrefixProps & NavbarProps> = ({ pathPrefix, consolesLin
     <Navbar className="mb-3" dark color="dark" expand="md" fixed="top">
       <NavbarToggler onClick={toggle} />
       <Link className="pt-0 navbar-brand" to={`${pathPrefix}/new/graph`}>
+        <img src="./prometheus_logo_grey.svg" alt="Prometheus logo" title="Prometheus" />
```

Comment:
Try adding `class="d-inline-block align-top"` (like in https://getbootstrap.com/docs/4.0/components/navbar/#brand) to vertically center the logo with the text.

Reply:
And a `margin-top: 4px;` on the entire Link around it to make the logo+text combo more centered with regards to the other menu items.